templates/partials/person_table.html

```diff
@@ -7,28 +7,29 @@
       {% if not entity.is_department and department_salaries %}
       <th scope="col">Department</th>
       {% endif %}
-      <th scope="col" class="text-right">Salary</th>
+      <th scope="col" class="text-right">Base Salary</th>
+      <th scope="col" class="text-right">Extra Pay</th>
       {% if show_start_date %}
       <th scope="col" class="text-right">Start Date</th>
       {% endif %}
     </tr>
   </thead>
   <tbody>
-  {% for job in jobs %}
+  {% for salary in salaries %}
     <tr>
-      <td><a href="{{ url('person', kwargs={'slug': job.person.slug}) }}">{{ job.person }}</a></td>
-      <td>{{ job.position.title }}</td>
+      <td><a href="{{ url('person', kwargs={'slug': salary.job.person.slug}) }}">{{ salary.job.person }}</a></td>
+      <td>{{ salary.job.position.title }}</td>
       {% if not entity.is_department and department_salaries %}
-      <td><a href="{{ url('department', kwargs={'slug': job.position.employer.slug}) }}">{{ job.position.employer }}</a></td>
+      <td><a href="{{ url('department', kwargs={'slug': salary.job.position.employer.slug}) }}">{{ salary.job.position.employer }}</a></td>
       {% endif %}
-      {% if job.salary %}
+      {% if salary.amount %}
         {# Use the prefetched salary. #}
-        {% set salary_at_hand = job.salary[0].amount %}
+        {% set salary_at_hand = salary.amount %}
       {% else %}
-        {% set salary_at_hand = job.salaries.get().amount %}
+        {% set salary_at_hand = salary.job.salaries.get().amount %}
       {% endif %}
       <td class="text-right">
-        {{ salary_at_hand|format_salary }}
+        ${{ salary_at_hand }}
```

Comment:
Instead of referencing `salary_at_hand` here, you can reference `salary.amount` directly.